openunmix/model.py

```diff
@@ -103,23 +129,39 @@ def istft(
             shape (nb_samples, nb_channels, nb_timesteps)
 
     """
-
-    shape = X.size()
-    X = X.reshape(-1, shape[-3], shape[-2], shape[-1])
-
-    y = torch.istft(
-        X,
-        n_fft=n_fft,
-        hop_length=n_hop,
-        window=window,
-        center=center,
-        normalized=False,
-        onesided=True,
-        length=length
-    )
-
-    y = y.reshape(shape[:-3] + y.shape[-1:])
-
+    if use_asteroid:
+        idft_filters = torch_stft_fb.TorchSTFTFB(
+            n_filters=n_fft,
+            kernel_size=n_fft,
+            stride=n_hop,
+            window=torch.hann_window(n_fft).numpy(),
+            center=center,
+            pad_mode="reflect",
+            normalized=False,
+            onesided=True,
+            sample_rate=44100.0,
```

Comment:
sample rate shouldn't be fixed

Reply:
I add this  in the params of the __ini__() then